diff -Naur kodi-18.0rc4-Leia.orig/xbmc/dialogs/GUIDialogContextMenu.cpp kodi-18.0rc4-Leia/xbmc/dialogs/GUIDialogContextMenu.cpp
--- kodi-18.0rc4-Leia.orig/xbmc/dialogs/GUIDialogContextMenu.cpp	2019-02-14 22:28:43.563579387 +0000
+++ kodi-18.0rc4-Leia/xbmc/dialogs/GUIDialogContextMenu.cpp	2019-02-14 22:31:00.159843386 +0000
@@ -241,7 +241,7 @@
       if (!share->m_ignore && !isAddon)
         buttons.Add(CONTEXT_BUTTON_REMOVE_SOURCE, 522); // Remove Source
 
-      buttons.Add(CONTEXT_BUTTON_SET_THUMB, 20019);
+    //  buttons.Add(CONTEXT_BUTTON_SET_THUMB, 20019);
     }
     if (!GetDefaultShareNameByType(type).empty())
       buttons.Add(CONTEXT_BUTTON_CLEAR_DEFAULT, 13403); // Clear Default
diff -Naur kodi-18.0rc4-Leia.orig/xbmc/favourites/GUIDialogFavourites.cpp kodi-18.0rc4-Leia/xbmc/favourites/GUIDialogFavourites.cpp
--- kodi-18.0rc4-Leia.orig/xbmc/favourites/GUIDialogFavourites.cpp	2019-02-14 22:28:43.563579387 +0000
+++ kodi-18.0rc4-Leia/xbmc/favourites/GUIDialogFavourites.cpp	2019-02-14 22:29:48.932247387 +0000
@@ -116,7 +116,7 @@
   }
   choices.Add(3, 15015); // Remove
   choices.Add(4, 118); // Rename
-  choices.Add(5, 20019); // Choose thumbnail
+  //choices.Add(5, 20019); // Choose thumbnail
 
   CFileItemPtr itemPtr = m_favourites->Get(item);
 
